
Sun, 26 Dec 2010 20:48:30 GMT

COMMAND LINE:
/usr/local/src/pypy-1.4.1-linux/bin/pypy spectralnorm.pypy-2.pypy 500

PROGRAM FAILED 


PROGRAM OUTPUT:

Traceback (most recent call last):
  File "app_main.py", line 53, in run_toplevel
  File "spectralnorm.pypy-2.pypy", line 11, in <module>
    from numpy import *
ImportError: No module named numpy
